client/my-sites/backup/backup-storage-space/backup-storage-space-upsell/style.scss

```
@@ -0,0 +1,42 @@
+.backup-storage-space-upsell__title-text {
+	font-size: 1rem;
+	font-weight: bold;
+	margin-bottom: 16px;
+}
+
+a.backup-storage-space-upsell__button {
+	color: #1e1e1e;
+}
+
+.backup-storage-space-upsell__button {
+	border: 2px solid var( --studio-jetpack-green );
```

It might be a good idea to use colors that derives from the current theme instead of referencing Jetpack Green explicitly, since Calypso Blue still has its own Backup page. `--color-primary`, maybe?

Pinging @Automattic/jetpack-design for their feedback here.

```suggestion
	border: 2px solid var( --color-primary );
```